stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -573,6 +562,22 @@ public final class _ExecutionContext {
   /// The mutex for preventing potential concurrent access.
   private var mutex: pthread_mutex_t = pthread_mutex_t()
 
+  /// List of devices available to this execution context.
+  /// Devices are represented by their names in TensorFlow notation.
+  /// See documentation for the top-level `withDevice` function for details
```

Comment:
Swift functions are identified by their full name, which includes argument labels. Therefore, we often use the full name in doc comments.

```suggestion
  /// See documentation for the top-level `withDevice(_:perform:)` function for details
```